apps/src/applab/applab.js

```diff
@@ -1240,6 +1242,10 @@ Applab.execute = function() {
   divApplab.firstChild.focus();
 
   Applab.running = true;
+  $('#headers').addClass('readonly');
+  $('#codeWorkspace').addClass('readonly');
+  // TODO - only if designing?
```

Comment:
we probably shouldn't do this if `level.hideDesignMode`. You can try it on https://studio.code.org/s/cspunit3/stage/2/puzzle/3 to see if it works in that case.